modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -28,6 +28,39 @@ Older versions may work, but are not supported.
 
 The most current and up to date API Documentation is always available through the https://pkg.go.dev/github.com/couchbase/gocb/v2[godoc website].
 
+== Version 2.3.4 (16 November 2021)
+
+Version 2.3.4 is a maintenance release for the Go SDK 2.3.0.
+
+https://pkg.go.dev/github.com/couchbase/gocb/v2@v2.3.4?tab=doc[API Documentation]
+
+=== New Features and Behavioral Changes
+
+* https://issues.couchbase.com/browse/GOCBC-1179[GOCBC-1179]:
+Added support to attempt graceful closing of connections.
+
+* https://issues.couchbase.com/browse/GOCBC-1154[GOCBC-1154];
+https://issues.couchbase.com/browse/GOCBC-1184[GOCBC-1184]:
+Added RateLimitFailure and QuotaLimitFailure support for Couchbase Capella.
+
+* https://issues.couchbase.com/browse/GOCBC-1193[GOCBC-1193]:
+Added the ResponseBody field to QueryError and N1qlError, to allow easier debugging and error handling.
+
+=== Fixed Issues
+
+* https://issues.couchbase.com/browse/GOCBC-1185[GOCBC-1185]:
+Fixed an issue with Queue and Set retrying during pop and remove operations.
+
+* https://issues.couchbase.com/browse/GOCBC-1186[GOCBC-1186]:
+Fixed issue where logging meter could cause a deadlock on closing the cluster object.
+
+* https://issues.couchbase.com/browse/GOCBC-1187[GOCBC-1187]:
+Fixed issue where logging meter could log a service/operation pair which has no operations.
+
+* https://issues.couchbase.com/browse/GOCBC-1194[GOCBC-1194]:
+Changed ordering of route config bootstrapping, to check all seed nodes for the default network type first.
+This fixed an issue with stuck deployments after upgrade to server 7.0.2.
```

Comment:
Unsure we should include this line?

Reply:
modified after consultation with @RichardSmedley